File: April 3, 2026.py
def get_browser_history(commands):
    history = []
    current = -1

    for command in commands:
        if command == "Back":
            if current > 0:
                current -= 1

        elif command == "Forward":
            if current < len(history) - 1:
                current += 1

        else:
            history = history[:current + 1]
            history.append(command)
            current += 1

    return [history, current]

File: April 3, 2026.js
function getBrowserHistory(commands) {

    const history = [];
    let current = -1;

    for (let command of commands) {
        if (command == "Back") {
            if (current > 0) {
                current--;
            }
        } else if (command == "Forward") {
            if (current < history.length - 1) {
                current++;
            }
        } else {
      
        history.splice(current + 1);
      
        history.push(command);
        current++;
    }
  }

  return [history, current];
}